pibooth/camera/base.py

```diff
@@ -9,14 +9,18 @@
 
 class BaseCamera(object):
 
-    def __init__(self, resolution, delete_internal_memory):
+    def __init__(self, iso, resolution, delete_internal_memory):
         self._cam = None
         self._border = 50
         self._window = None
         self._overlay = None
         self._captures = []
         self.resolution = resolution
         self.delete_internal_memory = delete_internal_memory
+        if not isinstance(iso, (tuple, list)):
+            iso = (iso, iso)
+        self.iso_preview = iso[0]
+        self.iso_capture = iso[1]
```

Comment:
Up to you but we can change it to:
```suggestion
        self.iso_preview, self.iso_capture = iso
```